onchain/rollups/test/foundry/dapp/Application.t.sol

```diff
@@ -944,11 +950,7 @@ contract ApplicationTest is TestBase {
         for (uint256 i; i < proofs.length; ++i) {
             LibServerManager.Proof memory proof = proofs[i];
             if (proof.proves(outputEnum, inputIndexWithinEpoch, outputIndex)) {
-                return
-                    Proof({
-                        validity: _convert(proof.validity),
-                        inputRange: inputRange
-                    });
+                return _convert(proof.validity, inputRange);
```

Comment:
`LibServerManager` would be updated after the server manager updates its response content, right?

Reply:
Yes, there are just artifacts that lets us test the new output validation scheme using the old machine.